ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/LoginSteps.cs

```diff
@@ -44,6 +45,7 @@ public void WhenTheUserAttemptsToLogout()
         public void WhenTheUserSignsOut()
         {
             WhenTheUserAttemptsToLogout();
+            Thread.Sleep(5000);
```

Comment:
Could you remove thread.sleep

Reply:
I've modified to now use **_browsers[_c.CurrentUser].Driver.WaitUntilVisible(...)** to assert the sign out message, whenever it appears.